src/utils/Downloader.js

```diff
@@ -16,6 +16,12 @@ const fetchAPI = require('@adobe/helix-fetch');
 
 const { TimeoutError } = fetchAPI;
 
+const DEFAULT_FORWARD_HEADERS = [
```

Comment:
why is this here? can you merge/rebase master again?

Reply:
This is here because I rebased to `master`. Doing it again won't help.

Reply:
well, that last `pull` destroyed your rebase again... no?
also, I would squash all commits first, before rebase with master, then you only need to resolve the conflicts once.